lib/engine/round/base.rb

```diff
@@ -130,6 +130,10 @@ def active_step(entity = nil)
         @active_step ||= @steps.find { |step| step.active? && step.blocking? }
       end
 
+      def derived_actions
```

Comment:
do we need derived actions? can't we just do @game.add_x_action(action)

def add_x_action(action)
  raise if @loading

Reply:
as discussed, auto_actions need to be able to be doable on a transition between steps so this is probably the cleanest way of doing it.